packages/common/src/hooks/useHostContext.ts

```diff
@@ -0,0 +1,21 @@
+import create from 'zustand';
+import { LocaleKey, SupportedLocales } from '../intl/intlHelpers';
+
+type HostContext = {
+  locale: SupportedLocales;
+  titleKey?: LocaleKey;
+  setLocale: (locale: SupportedLocales) => void;
+  setTitleKey: (titleKey: LocaleKey) => void;
```

Comment:
Also have realised.. the title should be `breadcrumbs`? https://material-ui.com/components/breadcrumbs/

So.. maybe should be a `useBreadcrumbs` hook? Similar to the title one but possibly will need a way to append rather than set, and also a way to reset?

Reply:
yeah, a little unclear from the designs. I'll have another go.

Reply:
hehehe.. have gone with a different approach. I rather like it: now using the navigation to drive the breadcrumb rather than have the pages themselves push their title.
d166be8
300f61e